lib/server.ts

```diff
@@ -96,8 +95,9 @@ export function startDevServer(srcs: string[], dists: string[], devJsonPath: str
  * (separated out to enable endpoint testing)
  * @param opts 
  * @param devJsonPath 
+ * @param gateway
  */
-export function createApp(devJsonPath: string, opts: DevOptions): Express.Application {
+export function createApp(devJsonPath: string, opts: DevOptions, gateway: GatewayConfigObject = DEFAULT_GATEWAY): Express.Application {
```

Comment:
We don't need the = DEFAULT _GATEWAY since we know startDevServer already has this fallback